specs/Device/device-schema.json

```diff
@@ -83,7 +83,8 @@
             "lte-m",
             "cat-m",
             "3g",
-            "grps"
+            "grps",
+            "coap"
```

Comment:
COAP is not a connectivity technology, are you sure it fits here?

Reply:
I wasn't sure since I see both `http` and `3g`.  I have removed that commit for now.

Reply:
@brahma-dev you were right, `coap` is a perfectly valid value for that attribute. you can restore it, thanks!

Reply:
@jmcanterafonseca Alright. I have made the necessary changes. Please have a look. Also fixed the typo in `gPRs`